tests/iface_namingmode/test_iface_namingmode.py

```diff
@@ -709,14 +757,18 @@ def test_show_acl_table(setup, setup_config_mode, tbinfo):
             elif mode == 'default':
                 assert item in acl_table
 
-def test_show_interfaces_neighbor_expected(setup, setup_config_mode, tbinfo):
+def test_show_interfaces_neighbor_expected(setup, setup_config_mode, tbinfo,duthosts, enum_rand_one_per_hwsku_frontend_hostname):
     """
     Checks whether 'show interfaces neighbor expected' lists the
     interface names as per the configured naming mode
     """
     if tbinfo['topo']['type'] not in ['t1', 't2']:
         pytest.skip('Unsupported topology')
 
+    duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
```

Comment:
can you use the new fixture you wrote to skip masic patform.

Reply:
Changed to use common function